packages/datagrid/src/components/default-header-renderer/header-grid.component.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import QualityBar from './quality-bar.component';
+import theme from './header-grid.scss';
+import {
+	TALEND_QUALITY_KEY,
+	TALEND_QUALITY_INVALID_KEY,
+	TALEND_QUALITY_EMPTY_KEY,
+	TALEND_QUALITY_VALID_KEY,
+} from '../constants';
+
+export const HEADER_RENDERER_COMPONENT = 'headerGrid';
+
+export default function DefaultHeaderRenderer({ column, displayName, onFocusedColumn }) {
```

Comment:
The folder name doesn't fit with component Name.
Components/DefaultHeaderRenderer/DefaultHeaderRenderer.component.js